README.md

```diff
@@ -192,6 +192,26 @@ This is the only harness that uses `run_benchmark`'s argument, `num_itrs_hint`.
 ./run_benchmarks.rb --rss
 ```
 
+## Rendering a graph
+
+`--graph` option of `run_benchmarks.rb` allows you to render benchmark results as a graph.
+
+```
+./run_benchmarks.rb --graph
```

Comment:
Should document where people should expect to find the generated image file.

Reply:
addressed in https://github.com/Shopify/yjit-bench/commit/cf535036b263350f4b55e58d2270570ed5c25e86